v63004/inref/gtm8922.m

```diff
@@ -13,55 +13,58 @@
 generalTest
 	SET dbFlush="DBFLUSH n_dsk_write"
 	SET dbSync="DBSYNC n_db_fsync"
-	SET jnlFlush="JNLFLUSH n_jnl_flush JNLFLUSH n_jnl_fsync"
-	SET epoch="EPOCH n_db_flush EPOCH n_jrec_epoch_regular"
-	SET flush="FLUSH n_db_flush FLUSH n_jrec_epoch_regular"
+	SET jnlFlush="JNLFLUSH n_jnl_flush|n_jnl_fsync"
+	SET epoch="EPOCH n_db_flush|n_jrec_epoch_regular"
+	SET flush="FLUSH n_db_flush|n_jrec_epoch_regular"
 
 	; the params string takes the following form
 	; "<VIEW OPTIONS> <STAT ARG>[ <VIEW OPTIONS> <STAT ARG>...]
 	SET params=dbFlush_" "_dbSync_" "_jnlFlush_" "_epoch_" "_flush
 
-	;loops throug words in params to call VIEW on each option and showBuffer() on each stat arg
+	;loops through words in params to call VIEW on each option and showBuffer() on each stat arg
 	FOR I=1:1:$L(params," ") DO
 	. SET keyword=$P(params," ",I) ; option passed to VIEW
-	. SET stat=$P(params," ",I+1)  ; register to peek with showBuffer
+	. SET stats=$P(params," ",I+1)  ; PEEKBYNAME field names to use in showBUFFER to verify VIEW operation
 	. WRITE "TESTING KEYWORD: "_keyword,!
-	. WRITE "------------------------",!
+	. WRITE "--------------------------",!
 	. SET ^DefTmp="MOE"_I
 	. SET ^AregTmp="LARRY"_I
 	. SET ^BregTmp="CURLY"_I
-	. DO showBuffer(stat)
+	.
+	. FOR J=1:1:$L(stats,"|") DO  ;stats is a list of multiple '|' delimited fields
+	. . DO showBuffer($P(stats,"|",J))
 	. WRITE "VIEW """_keyword_""":""AREG,BREG,BREG""",!
 	. VIEW keyword:"AREG,BREG,BREG"
```

Comment:
Can we have BREG,BREG,AREG instead of AREG,BREG,BREG here. I want at least one usage where BREG comes before AREG.